pkg/karmadactl/karmadactl.go

```diff
@@ -47,7 +47,7 @@ func NewKarmadaCtlCommand(out io.Writer, cmdUse, cmdStr string) *cobra.Command {
 	rootCmd.AddCommand(sharedcommand.NewCmdVersion(out, cmdStr))
 	rootCmd.AddCommand(NewCmdCordon(out, karmadaConfig, cmdStr))
 	rootCmd.AddCommand(NewCmdUncordon(out, karmadaConfig, cmdStr))
-	rootCmd.AddCommand(NewCmdGet(out, karmadaConfig))
+	rootCmd.AddCommand(NewCmdGet(out, karmadaConfig, cmdStr))
```

Comment:
I suggest rename this `cmdStr` to `parentCommand` as well.

Reply:
Already modified. PTAL